pkg/execution/driver/httpdriver/httpdriver.go

```diff
@@ -272,6 +274,8 @@ func (e executor) do(ctx context.Context, url string, input []byte) (*response,
 		return nil, fmt.Errorf("error reading response body: %w", err)
 	}
 
+	hv, _ := strconv.Atoi(resp.Header.Get("x-inngest-req-version"))
```

Comment:
Can we log when their's an `atoi` error?

Reply:
Nope.  If there's no version, this means "0" which is explicitly different to unset.